components/inference-service/monaiinference/handler/kubernetes.py

```diff
@@ -182,29 +183,64 @@ def __build_kubernetes_persistent_volume_claim(self) -> models.V1PersistentVolum
     def create_kubernetes_pod(self):
         """Create a kubernetes pod and the Persistent Volume and Persistent Volume Claim needed by the pod.
         """
-        # Create a Kubernetes Persistent Volume.
-        pv = self.__build_kubernetes_persistent_volume()
-        self.kubernetes_core_client.create_persistent_volume(pv)
-
-        # Create a Kubernetes Persistent Volume Claim.
-        pvc = self.__build_kubernetes_persistent_volume_claim()
-        self.kubernetes_core_client.create_namespaced_persistent_volume_claim(namespace=DEFAULT_NAMESPACE, body=pvc)
-
-        # Create a Kubernetes Pod.
-        pod = self.__build_kubernetes_pod()
-        self.kubernetes_core_client.create_namespaced_pod(
-            namespace=DEFAULT_NAMESPACE,
-            body=pod
-        )
+
+        try:
+            # Create a Kubernetes Persistent Volume.
+            pv = self.__build_kubernetes_persistent_volume()
+            self.kubernetes_core_client.create_persistent_volume(pv)
+            logging.info(f'Created Persistent Volume {pv.metadata.name}')
+        except Exception as e:
+            logging.error(traceback.format_exc())
```

Comment:
How about using `exe_info` parameter instead of getting traceback information from traceback module?
(Currently, it would lose error message from the Exception/Error).

```suggestion
            logging.error(e, exc_info=True)
```
And, I think it is better to avoid using a root logger.
```python
logger = logging.getLogger(__name__)
...

def foo():
    try:
        ...
    except XXXError as e:
        logger.error(e, exc_info=True)
```